tests/cases/fourslash/goToDefinitionSwitch1.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference path="fourslash.ts" />
+
+/////*end*/switch (null) {
+////  [|/*start*/case|] null: break;
```

Comment:
I don't think this needs a `[|range|]` does it?

Reply:
Please remove these from the tests if they're not necessary

Reply:
```

Reply:
You mean like this? It doesn't work.

```
/////*end*/switch (null) {
////  /*start*/case null: break;
////}
```

Reply:
Really? That's very weird, but I'm only a little surprised. 😅

Reply:
the tests from the previous pull request are taken as a basis
https://github.com/microsoft/TypeScript/blob/main/tests/cases/fourslash/goToDefinitionReturn1.ts

Reply:
This test passes:
```
/////*end*/switch (null) {
////  [|/*start*/case|] null: break;
////}
```

But this:
```
/////*end*/switch (null) {
////  /*start*/case null: break;
////}
```
Give error: 
```
     Error: At marker 'start': goToDefinitionsAndBoundSpan failed. Found a starting TextSpan around 'case' in '/tests/cases/fourslash/goToDefinitionSwitch1.ts' (at position 18). If this is the correct input span, put a fourslash range around it: 

      switch (null) {
        [|/*start*/case|] null: break;
      }

```